software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -76,6 +76,10 @@
     @CatalogConfig(label = "Stop Command", priority=1)
     ConfigKey<String> STOP_COMMAND = VanillaSoftwareProcess.STOP_COMMAND;
 
+    @CatalogConfig(label = "Suppress failures occurred during installation. When using this parameter, Apache Brooklyn relies that your blueprint will report failures on the check running phase.")
+    ConfigKey<Boolean> SUPPRESS_INSTALL_FAILURE = ConfigKeys.newBooleanConfigKey("install.windows.suppressFailure", "Suppress any installation failure. It is a common case to restart windows together with the installation process." +
```

Comment:
I think we should indicate what the default value is (`false` I presume), rather than leaving it null.